pkgs/development/libraries/netcdf/default.nix

```diff
@@ -60,7 +60,7 @@ in stdenv.mkDerivation rec {
     remove-references-to -t ${stdenv.cc} "$(readlink -f $out/lib/libnetcdf.settings)"
   '';
 
-  doCheck = !mpiSupport;
+  doCheck = !(mpiSupport || stdenv.isDarwin);
```

Comment:
The tests seems to pass on x86_64-darwin, so that's probably better to disable them on aarch64-darwin only:

```
doCheck = !(mpiSupport || (stdenv.isDarwin && stdenv.isAarch64));
```